cranelift/assembler-x64/meta/src/dsl/custom.rs

```diff
@@ -0,0 +1,85 @@
+use core::fmt;
+use std::ops::BitOr;
+
+#[derive(PartialEq, Debug)]
+pub enum CustomOperation {
+    Visit,
+    Display,
+    None,
```

Comment:
```suggestion
    Encode,
```

We don't need `None` here because `Custom` can be empty.